### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by sakinaalimsft**
Addition of detailed instructions for how to install WSL as a prerequisite to toolchain configuration and other edits for clarity.

### CODE REVIEW

**Comment by sakinaalimsft** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: docs/deploy/index.md [hunk 1]
```diff
@@ -144,18 +144,52 @@ az vm image terms accept --offer almalinux-hpc --publisher almalinux --plan 8_5-
...
 ## From a local machine
-The toolchain is built to be run only from Linux and preferably Ubuntu 20.04+. So if you are running a Windows desktop you should use WSL2 with an Ubuntu 20.04 environment. All the following commands will have to be run from your Linux shell terminal.
+
+The toolchain for deployment is built to be run only from Linux, and preferably Ubuntu 20.04+. So, if you are running a Windows desktop you should use WSL2 with an Ubuntu 20.04 environment. Run the following commands to install WSL2 (you will only need to do this once on your local device).
+
+To install WSL2 in your Windows machine:
+
+1.	Open PowerShell as administrator.
+
+ ![image](https://user-images.githubusercontent.com/119871043/219146537-14e9a00f-9c56-4f51-b566-e715e520cd2a.png)
```
Comment (xpillons) [inline docs/deploy/index.md:154]: Please make sure to add images as well in the repo and not outside

**Review by xpillons — CHANGES_REQUESTED** [review]
please add the images too in the repo, not only the text.

**Comment by sakinaalimsft** [comment]
Made above changes to text to point to images in repo under azhop/docs/images as requested

**Review by xpillons — APPROVED** [review]

**Pull request merged**